packages/backend-deployer/src/cdk_deployer.ts

```diff
@@ -169,6 +169,11 @@ export class CDKDeployer implements BackendDeployer {
       aggregatedStderr += chunk;
       done();
     };
+
+    // Piping the output by default strips off the color. This is a workaround to
+    // preserve the color being piped to parent process. Later we reset it to whatever it was before.
+    const oldForceColorValue = process.env['FORCE_COLOR'];
+    process.env['FORCE_COLOR'] = '1';
```

Comment:
Can we clone `process.env`, append this to a copy and pass to child process ?

Reply:
Good idea, only setting in the child process now.